bash_scripting_101/bash_scripting_101.md

```diff
@@ -37,241 +39,443 @@ script: https://kit.fontawesome.com/83b2343bd4.js
+
+From your `learning_bash` directory create a new file called `my_file`:
+
+```
+cd ~/learning_bash
+touch my_file
+```
+
+You can also create multiple files at a time by listing their unique names one after another:
+
+```
+cd ~/learning_bash
+touch file_1 file_2 file_3
+```
+
+Now if you use `ls` to see the contents of `learning_bash`, you should see four files: `file_1`, `file_2`, `file_3`, and `my_file`.
+
+Since these are files and not directories, we can't use `cd` to open them, and we didn't put anything into these files yet anyway. We will learn how to interact with their contents in the next section.
+
+
+<div class = "warning">
+You might have noticed that none of the file or directory names we used had spaces in them. Bash scripting gets quite a bit more complicated when you have spaces in names, so it is considered best practice to avoid them.
 </div>
-***********************************************************************
 
-## Permissions and working with other users
+### Writing to a file directly
+
+We have created 4 files, but so far all of them are empty. The most direct way to write text to a file is to use the `echo` and `>>` commands.
+
+**`echo`**
+
+The bash command `echo` returns any text that follows it. Give it a try in your command line interface!
+
+**`>>`**
 
-- `Chmod` or change mode is used to set the permissions on a file or directory.
-- The three basic permissions are:
+The bash command `>>` is used to write the output of a command to the end of a file. For example the following code will create a file called `my_sentences` and write the text `This is a sentence.` in that file.
 
-  - **read** - access the contents of a file
-  - **write** - modify the contents of a file or directory
-  - **execute** - run a bash script on a file or directory.
+```
+echo "This is a sentence." >> my_sentences
+```
 
-- [Consult this documentation and table for guidance on how to apply useful file and directory level permissions](https://linuxcommand.org/lc3_lts0090.php)
-- `Chown`: change file ownership
-- `Chgrp`: change group ownership
-- `Sudo` - assume super user privileges on an as-needed basis.
+If you then want to add an additional line to `my_sentences` you can use the same method.
+
+```
+echo "This is another sentence." >> my_sentences
+```
+
+Now the file `my_sentences` has two lines:
+
+```
+This is a sentence.
+This is another sentence.
+```
+
+**Give it a try!** Add some text to the files you already created!
 
 <div class = "warning">
-**Proceed with caution!** Using the `Sudo` command is decidedly not best data security practice, but might need to be invoked as you continue to learn bash, as permissioning can be quite granular and dynamic.
+Make sure you use the two arrows in `>>`. Using only a single arrow will also write text to your file, but instead of appending it to the end of the file, it will replace the entire contents of the file with that new text.
```

Comment:
```suggestion
Make sure you use the two greater than signs (here, we'll call them "arrows") in `>>`. Using only a single arrow will also write text to your file, but instead of appending it to the end of the file, it will replace the entire contents of the file with that new text.
```